### PR TITLE
Create Test2.java

### PR DESCRIPTION
**Pull request by GokhanRepo**
More test file

### CODE REVIEW

File: src/Test2.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// some code
```
Comment (GokhanRepo) [inline src/Test2.java]: Test comment via web editor

Comment (evadamtk) [reply src/Test2.java]: all well done

**Comment by evadamtk** [comment]
show must go on...

File: src/Test2.java (hunk 1 above)
Comment (GokhanRepo) [reply src/Test2.java]: Thank you. Have a good one 😄

**Pull request merged**